modules/lz/sub-vending/src/self/subResourceWrapper/deploy.bicep

```
@@ -110,6 +113,9 @@ var virutalWanHubDefaultRouteTableId = {
 var virtualWanHubConnectionPropogatedRouteTables = !empty(virtualNetworkVwanPropagatedRouteTablesResourceIds) ? virtualNetworkVwanPropagatedRouteTablesResourceIds : array(virutalWanHubDefaultRouteTableId)
 var virtualWanHubConnectionPropogatedLabels = !empty(virtualNetworkVwanPropagatedLabels) ? virtualNetworkVwanPropagatedLabels : [ 'default' ]
 
+// Telemetry for CARML flip
+var carmlTelemetryFlip = disableTelemetry == true ? false : true
```

I get the intent, but wouldn't it be more intuitive to name the variable as per its meaning instead of its function?
```suggestion
var enableTelemetryForCarml = disableTelemetry == true ? false : true
```